### ISSUE TITLE
Integration with Authelia

### ISSUE THREAD

**Issue by arbdevml** [wushilin/minica#3]
Hello, first of all very big thank you for this project.
Is it possible to add the option login from the header (Trusted Header SSO, https://www.authelia.com/integration/trusted-header-sso/introduction/)?
Trying to find a way to integrate this project with Authelia.
Thank you for your answer.

**Comment by wushilin** [comment]
Sure I would look into this!

**Comment by wushilin** [comment]
Fixed in v1.0.3

**Comment by wushilin** [comment]
Now it supports alternative authentication method. see release notes for options.

Example:

```
authentication.mode=request-header
request-header.name.username=x-user
#request-header.name.group=x-group
request-header.group.admin.name=$any
request-header.group.viewer.name=$any
```
